tools/README.md

```diff
@@ -339,25 +339,25 @@ Shuffle Skew Check: (When task's Shuffle Read Size > 3 * Avg Stage-level size)
 #### C. Health Check
 
 ```
-Application application_1616746343401_0025 (index=1) failed tasks:
-+-------+--------------+------+-------+------------------------------------+
-|stageId|stageAttemptId|taskId|attempt|endReason_first36char               |
-+-------+--------------+------+-------+------------------------------------+
-|4      |0             |2842  |0      |ExceptionFailure(ai.rapids.cudf.Cudf|
-|4      |0             |2858  |0      |TaskKilled(another attempt succeeded|
-|4      |0             |2884  |0      |TaskKilled(another attempt succeeded|
-|4      |0             |2908  |0      |TaskKilled(another attempt succeeded|
-|4      |0             |3410  |1      |ExceptionFailure(ai.rapids.cudf.Cudf|
-+-------+--------------+------+-------+------------------------------------+
-
-Application application_1616746343401_0025 (index=1) failed stages:
+Failed tasks:
++-------+--------------+------+-------+----------------------+
+|stageId|stageAttemptId|taskId|attempt|endReason_first100char|
++-------+--------------+------+-------+----------------------+
+|      4|             0|  2842|      0|  ExceptionFailure(...|
```

Comment:
this doesn't look like 100 chars?

Reply:
Yes, For readibility purpose had truncated the output.  I will update correct output in the README and we can decide if it's okay to keep it.